### ISSUE TITLE
Recompute some MSA while --use_precomputed_msas flag is on

### ISSUE THREAD

**Issue by marekb-sci** [deepmind/alphafold#469]
The HHSearch is run also with --use_precomputed_msas flag enabled
https://github.com/deepmind/alphafold/blob/b85ffe10799ca08cc62146f1dabb4e4ee6c0a580/alphafold/data/pipeline.py#L184-L196

Bug or feature?

**Comment by Augustin-Zidek** [comment]
It is a feature since running HHSearch takes a lot of time so it is useful being able to reuse its result from previous runs for the same sequence (given that nothing changed in the data pipeline and the sequence and the MSAs are identical, since HHSearch is deterministic given its sequence and MSA input).

**Comment by marekb-sci** [comment]
Maybe my post was unclear, let me rephrase it: even with `--use_precomputed_msas` hhsearch query IS RUNNED. This happen in the  lines I quoted and the flag seems to have no effect here. `pdb_template_hits` is recomputed no matter `--use_precomputed_msas` is on or off.

**Comment by Augustin-Zidek** [comment]
Ah yes, you are indeed right -- this applies only to MSA searches but not to the template search.

**Comment by marekb-sci** [comment]
any reason for that? In our fork we use saved file if `--use_precomputed_msas` is set
